### PR TITLE
feat: Added support for `deduplication_scope` and `fifo_throughput_limit`

### PR DESCRIPTION
**Pull request by kamilhristov**
## Description

1. Add more variables for fifo queues with default values as per documentation: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sqs_queue
2. Allow example to be applied without changes.

## Motivation and Context

Fixes https://github.com/terraform-aws-modules/terraform-aws-sqs/issues/31

## Breaking Changes

None.

## How Has This Been Tested?
- [x] I have tested and validated these changes using one or more of the provided `examples/*` projects

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -82,6 +82,18 @@ variable "kms_data_key_reuse_period_seconds" {
   default     = 300
 }
 
+variable "deduplication_scope" {
+  description = "Specifies whether message deduplication occurs at the message group or queue level"
+  type        = string
+  default     = "queue"
+}
+
+variable "fifo_throughput_limit" {
```
Comment (antonbabenko) [inline variables.tf:91]: Make sure that you also update `main.tf` and use these variables there as arguments:

https://github.com/terraform-aws-modules/terraform-aws-sqs/blob/master/main.tf#L1-L21

File: variables.tf [hunk 2]
```diff
@@ -82,6 +82,18 @@ variable "kms_data_key_reuse_period_seconds" {
...
+  description = "Specifies whether the FIFO queue throughput quota applies to the entire queue or per message group"
+  type        = string
+  default     = "perQueue"
```
Comment (antonbabenko) [inline variables.tf:94]: Keep default value for these variables to `null` because these arguments are optional.

**Review by antonbabenko — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (kamilhristov) [reply variables.tf:91]: fixed 🤦‍♂️

**Comment by kamilhristov** [comment]
@antonbabenko comments addressed, please review 🙏

**Review by antonbabenko — CHANGES_REQUESTED** [review]
Please update `versions.tf` to use Terraform 0.13.1 at least, and AWS provider when the feature you are adding was introduced.

Copy this - https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/versions.tf

**Comment by antonbabenko** [comment]
Good job! Here we go 🎉 

[v3.2.0](https://github.com/terraform-aws-modules/terraform-aws-sqs/releases/v3.2.0) has been just released.

**Pull request merged**